harness/determined/cli/experiment.py

```diff
@@ -780,18 +781,42 @@ def set_gc_policy(args: Namespace) -> None:
             )
         )
 
-    if args.yes or render.yes_or_no(
-        "Changing the checkpoint garbage collection policy of an "
-        "experiment may result\n"
-        "in the unrecoverable deletion of checkpoints.  Do you wish to "
-        "proceed?"
+    if (
+        args.yes
+        or args.test
+        or render.yes_or_no(
+            "Changing the checkpoint garbage collection policy of an "
+            "experiment may result\n"
+            "in the unrecoverable deletion of checkpoints.  Do you wish to "
+            "proceed?"
+        )
     ):
         patch_experiment(args, "change gc policy of", {"checkpoint_storage": policy})
         print("Set GC policy of experiment {} to\n{}".format(args.experiment_id, pformat(policy)))
     else:
         print("Aborting operations.")
 
 
+def get_fmted_checkpoints(unfmted_checkpoints: Any) -> Any:
```

Comment:
i think we should get rid of this entirely. there is no need for checkpoints to go through the strange/archaic format. open to disagreement.